docs/src/embeddings/default_embedding_functions.md

```diff
@@ -221,7 +221,7 @@ We support CLIP model embeddings using the open source alternative, [open-clip](
 This embedding function supports ingesting images as both bytes and urls. You can query them using both test and other images.
 
 !!! info
-    LanceDB supports ingesting images directly from accessible links.
+    LanceDB supports ingesting images,audio and video directly from accessible links.
```

Comment:
I think here ingesting images directly referred to Lance Having image extension type and not just that these embedding functions can generate embeddings directly from links. So, I'd be a little hesitant to say we support "ingesting" anything other than images directly. But I'd defer this part to @changhiskhan

Reply:
yeah makes sense , I will specify how users can add local paths instead of this